### PR TITLE
Add doc extraction dependencies

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Add doc extraction dependencies

#### Ticket Link
[MM-34801](https://mattermost.atlassian.net/browse/MM-34801)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,9 @@ RUN apk add --no-cache \
   netcat-openbsd \
   xmlsec-dev \
   tzdata \
+  wv \
+  poppler-utils \
+  tidyhtml \
```
Comment (cpanato) [inline build/Dockerfile:23]: why we are adding those things? this is needed to run mattermost?

Comment (jespino) [reply build/Dockerfile:23]: This are dependencies for the document extraction that is now part of mattermost document search.

**Review by cpanato — APPROVED** [review]

**Comment by amyblais** [comment]
@jespino Joram is off this week, probably another dev needs to review this. v5.35 will be based on Cloud April 21st release which is coming on Wednesday.

**Review by mgdelacroix — APPROVED** [review]

**Pull request merged**